packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -791,45 +792,70 @@ class MediaQuery extends InheritedWidget {
   /// The data from the closest instance of this class that encloses the given
   /// context.
   ///
-  /// You can use this function to query the size an orientation of the screen.
-  /// When that information changes, your widget will be scheduled to be
-  /// rebuilt, keeping your widget up-to-date.
+  /// You can use this function to query the size and orientation of the screen,
+  /// as well as other media parameters (see [MediaQueryData] for more
+  /// examples). When that information changes, your widget will be scheduled to
+  /// be rebuilt, keeping your widget up-to-date.
   ///
   /// Typical usage is as follows:
   ///
   /// ```dart
   /// MediaQueryData media = MediaQuery.of(context);
   /// ```
   ///
-  /// If there is no [MediaQuery] in scope, then this will throw an exception.
-  /// To return null if there is no [MediaQuery], then pass `nullOk: true`.
+  /// If there is no [MediaQuery] in scope, this will throw an exception in
+  /// release builds, and throw a [FlutterError] in debug builds.
+  ///
+  /// The deprecated `nullOk` argument is ignored, and will assert if set to true.
+  /// Use [maybeOf] instead of setting `nullOk` to true.
+  static MediaQueryData of(BuildContext context, {
+    @Deprecated(
+      'For cases where nullOk would be true, use MediaQuery.maybeOf instead'
```

Comment:
missing punctuation; this would lead to the string being "use MediaQuery.maybeOf insteadThis feature was deprecated". Needs a period and a space at the end of the first line.

Reply:
Fixed.